packages/combobox/README.md

```diff
@@ -21,8 +23,73 @@ When looking to leverage the `Combobox` base class as a type and/or for extensio
 import { Combobox } from '@spectrum-web-components/combobox';
 ```
 
-## Example
+## Examples
+
+<!--- what is the most minimal example possible? are any of the attributes required? -->
+<!--- Each example in the RFC includes a prepended sp-field-label. Is that required, even with the "label" attr? -->
+<!-- https://combobox--spectrum-web-components.netlify.app/storybook/?path=/story/combobox--default is empty -->
+
+```html
+<sp-combobox>
+    <sp-menu-item value="red">Red</sp-menu-item>
+    <sp-menu-item value="green">Green</sp-menu-item>
+    <sp-menu-item value="blue">Blue</sp-menu-item>
+</sp-combobox>
+```
+
+### Options Array
+
+Instead of providing `<sp-menu-item>` children, you can assign an array of `ComboboxOptions` to the `options` property, and `<sp-combobox>` will create matching menu items:
+
+<!--- Why does ComboboxOptions have "id" and "value" rather than "value" and "label" to match <option> & provide a data-value and a render-value? -->
+
+```html
+<sp-combobox id="colors"></sp-combobox>
+
+<script>
+    document.getElementById('colors').options = {[
+        { id: "red", value: "Red" },
+        { id: "green", value: "Green" },
+        { id: "blue", value: "Blue" }
+    ]};
+</script>
```

Comment:
I don't think this will work. @najikahalsema was making some similar updates to UEC that might be applicable here.

This however, points out that I need to add this to the `bundle` package so that it is even available in the docs pages like this... I'll get a separate PR up for that.

Reply:
Yeah, you'd probably want to do something like 

```
```html-live
<sp-combobox id="colors"></sp-combobox>

<script type="module">
    const initCombobox = () => {
        const combobox = document.querySelector('#colors');
        const options = [
            { id: "red", value: "Red"}, ......
         ];
       combobox.options = options;
    };
    customElements.whenDefined('sp-combobox').then(() => {
        initCombobox();
    });
</script>
```
And then immediately below it (but outside of the html-live tag):
```
<script type="module">
    const initCombobox = async () => {
        const combobox = document.querySelector('#colors');
        await combobox.updateComplete;
        await combobox.closest('code-example')?.updateComplete;

        const options = [
            { id: "red", value: "Red"}, ......
         ];
       combobox.options = options;
    };
    customElements.whenDefined('code-example').then(() => {
        customElements.whenDefined('sp-combobox').then(() => {
            initCombobox();
        });
    });
</script>
```

Reply:
Is this a place where it deviates from the RFC? [Using an object](https://github.com/adobe/spectrum-web-components/discussions/3897) looks to be assigning an array to the `options` property, just via Lit syntax.

Reply:
Or do you just mean this won't work in the docs demo environment?

Reply:
It won't work in the demo environment... maybe. Unless you find that it does! In which case, never mind!

Reply:
Thanks for the example!